rsync.exclude

```diff
@@ -6,3 +6,4 @@ ci_build_images
 docker-compose/logs
 docker-compose/mariadb
 master-private.cfg
+dockerfiles
```

Comment:
I may be wrong but this seems to be needed by one of the master (nonstandard or galera, not sure exactly which one).